src/features/shared/molecules/Pagination/Pagination.css

```diff
@@ -0,0 +1,12 @@
+.pagination-component {
+    margin: 1em;
+}
+.pagination-button {
+    padding: 10px;
+}
+.selected-pagination {
+    color: rgb(59 130 246 / 0.5);
+}
+.disabled-pagination {
+    color: #393e46;
+}
```

Comment:
I searched for a style reference (like a Figma template) that you use, but I didn't find anything. However, I can change it if you want to give me a model :)